Upstream-Status: Pending

Subject: Makefile.am: fix parallel build problem

Fix parallel build problem to avoid errors like below.

    install: cannot stat 'testsuite/module-playground/mod-fake-cciss.ko': No such file or directory

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 9457655..d5b02eb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -237,7 +237,7 @@ rootfs: build-module-playground
 
 .PHONY: rootfs build-playground
 
-$(ROOTFS): $(ROOTFS_PRISTINE)
+$(ROOTFS): $(ROOTFS_PRISTINE) build-module-playground
 	$(CREATE_ROOTFS)
 
 TESTSUITE_OVERRIDE_LIBS = \
-- 
1.9.1

